package.json

```diff
@@ -69,6 +69,7 @@
     "@protobuf-ts/runtime-rpc": "^2.9.4",
     "@uiw/codemirror-themes": "^4.23.3",
     "@uiw/react-codemirror": "^4.23.3",
+    "ace-builds": "1.36.2",
```

Comment:
Oh, you're right, it's no longer necessary. I needed to do it when I tried to talk to Ace editor using its own API in the test, but since I abandoned this approach, I can revert this.